COLLABORATOR_GUIDE.md

```diff
@@ -4,7 +4,15 @@
 
 * [Issues and Pull Requests](#issues-and-pull-requests)
 * [Accepting Modifications](#accepting-modifications)
- - [Involving the Website Admin Group](#involving-the-website-admin-group)
+  - [Involving the Website Admin Group](#involving-the-website-admin-group)
```

Comment:
```suggestion
  - [Involving the Website Group](#involving-the-website-group)
```

Reply:
Fixed, thank you for your quick reply :)